CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Multipage Review/ReviewViewController.swift

```diff
@@ -0,0 +1,415 @@
+//
```

Comment:
[Suggestion] Could you refactor/split this file so that the number of lines constraint of SwiftLint is met?